app/views/how-tos/switching-from-govuk-prototype-kit.html

```diff
@@ -0,0 +1,105 @@
+{% extends 'layout.html' %}
+
+{% block pageTitle %}
+  Switching from the GOV.UK prototype kit - NHS.UK prototype kit
+{% endblock %}
+
+{% block beforeContent %}
+  {% include "how-tos/includes/breadcrumb.html" %}
+{% endblock %}
+
+{% block content %}
+  <div class="nhsuk-grid-row">
+
+    <div class="nhsuk-grid-column-two-thirds">
+
+      <h1>
+        Switching from the GOV.UK prototype kit
+      </h1>
+
+
+<p>
+      If you have learned how to use the GOV.UK prototype kit, you will be able to use the NHS.UK prototype kit. This is because the two prototype kits are built in the same way. However, you must do some things differently.
```

Comment:
Link to it here as it’s the first mention (aside from the h1)? They also capitalise it unlike the NHS.

```suggestion
      If you have learned how to use the [GOV.UK Prototype Kit](https://prototype-kit.service.gov.uk/docs/), you will be able to use the NHS.UK prototype kit. This is because the two prototype kits are built in the same way. However, you must do some things differently.
```